src/build.sh

```diff
@@ -11,6 +11,18 @@ fi
 set -e                          # Abort on errors
 
 
+    
+# Define some environment variables
+export CC=${EXTERNAL_CC:-${CC}}
+export CXX=${EXTERNAL_CXX:-${CXX}}
+export F90=${EXTERNAL_F90:-${F90}}
+export LD=${EXTERNAL_LD:-${LD}}
+export CFLAGS=${EXTERNAL_CFLAGS:-${CFLAGS}}
+export CXXFLAGS=${EXTERNAL_CXXFLAGS:-${CXXFLAGS}}
+export F90FLAGS=${EXTERNAL_F90FLAGS:-${F90FLAGS}}
+export LDFLAGS=${EXTERNAL_LDFLAGS:-${LDFLAGS}}
```

Comment:
Having lots `EXTERNAL_CC` in the libraries is not my favorite solution to the issue of having to compile CarpetX codes with `nvcc` even though they are marked as `C++` sources to makefile, but, since I haven not been able to offer a better alternative, and with the reservation that these may go away in the (nopefully near) future: please apply so that there is a working set of codes for the hackathon (I assume these are needed for that)?

Reply:
Yes, these changes make things work for the hackathon. I have also updated my flesh branch `eschnett/crusher` to include the respective changes to make this work.